packages/manager/apps/telecom/src/app/telecom/telephony/alias/portability/portabilities/attach/portabilities-attach.controller.js

```diff
@@ -0,0 +1,47 @@
+import endsWith from 'lodash/endsWith';
+
+export default class {
+  /* @ngInject */
+  constructor($q, $timeout, $translate, OvhApiTelephony) {
+    this.$translate = $translate;
+    this.OvhApiTelephony = OvhApiTelephony;
+    this.$timeout = $timeout;
+    this.$q = $q;
+  }
+
+  $onInit() {
+    this.isLoading = false;
+    this.hasChecked = false;
+  }
+
+  uploadFile() {
+    this.isLoading = true;
+    return this.$q.all({
+      noop: this.$timeout(angular.noop, 5000),
```

Comment:
When I meant adding a bigger timeout, I meant on the resource call (see https://docs.angularjs.org/api/ngResource/service/$resource#$resource-arguments > actions > timeout) if it's needed